src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -292,6 +301,96 @@ message MethodConfigProto {
 
 }
 
+// `SampleValueSet` defines a set of parameter values used in
+// generating samples.
+message SampleValueSet {
+  // An identifier, unique to this method, for this set of values.
```

Comment:
maybe: "unique to a particular API method"

Reply:
done